### ISSUE TITLE
Use with Animate (Flash Professional) still not working

### ISSUE THREAD

**Issue by mc3dev** [myflashlab/webView-ANE#71]
Hi, I am using the demo version ANE and still have the problem with getting the RichWebView ANE to work in Flash Pro (Now Animate CC) right out of the bat. Here is the errors I got when I preview:

**Scene 1, Layer 'Layer 1', Frame 1, Line 1, Column 37	1172: Definition com.myflashlab.air.extensions:webView could not be found.
Scene 1, Layer 'Layer 1', Frame 1, Line 1, Column 37	1172: Definition com.myflashlab.air.extensions:webView could not be found.**

I used AIR 20.0 for Android on Animate and AIR 17.0 for Android for Flash Pro CC 2015, none of them work. 

In the closed issue a month ago you mentioned Episode 3 will show how the ANE work in Flash Pro. I have not found it on Youtube.

We really want to get this to work. Can you please advise? Thank you much.

**Comment by myflashlab** [comment]
We'll try to upload Episode 3 as soon as possible. but in the meanwhile. I suggest you to watch the other two episodes which will share a common set of issues.

also upgrade your Air SDK to at least Air 19.

**Comment by Dialoz** [comment]
Not sure how different the demo version is to the Commercial version but to help close this issue and for any other flash pro / animate cc people / users tying to use / test this ane.

I make apps exclusively using Adobe Animate CC (Formally Adobe Flash Pro CC)

To test a working example of richWebView I found I had to make sure to:
 
1.) - for any flash / animate users not familiar with using flex addon in flash / animate:
      Make sure to Download (Flex) : 
      http://blogs.adobe.com/flex/files/2012/05/FlexLicense.swf?build=4.5.1.21328A&pkgtype=3
      - and Extract & save flex sdk to you hard drive.
        then in Flash goto  'Edit' > 'Preferences' >'Compiler' and choose the folder you saved your flex sdk addon.

2.) most of you who have used ANE in Flash Pro / Animate but in case you forgot.
     from 'ActionScript Setting' > Click 'browse to native extension ane file.'  
     choose richWebView.ane > open > ok
     
     then go back into 'ActionScript Setting' > Click 'browse to SWC file.'
     choose  richWebView.swc > open > ok

3.) REMEMBER, this is native & this will not work when testing in flash / animate.
     however It will publish APK correctly and richWebView and when you test on your device all works correctly.

So in conclusion I have found this works great for Developers using Flash / Animate for their android and ios apps.    

Hope this helps anyone having issues testing.

**Comment by myflashlab** [comment]
@Dialoz Thanks for your input. We rarely use Flash IDE for mobile developments. FlashDevelop IDE has been our favorite for years. we hope this information can help other devs whom might have faced a similar problem.

Just one thing I should add is that, I don't think using the ```.swc``` file is really necessary as FlashCC or as it is called today, AnimateCC can read .ane files without the need to use swc files.

@mc3dev I'm closing the issue now as we haven't heard back from you. But if you still have any problem in this regard, please do feel free to contact us or re-open the issue... we're here to help.
